game/combo.go

```diff
@@ -64,6 +65,7 @@ type Player interface {
 
 type Game interface {
 	Play() Player
+	Board() Board
```

Comment:
What if we made the ending board a 2nd return value of Play()? Or maybe just have Play() return a new GameResult struct that includes various things like winner, pieces remaining, time spent (once tracked)? It's true we will need some way to get the board during the game eventually, but I'm thinking of adding a Spectator type that you can add to a game and it will get game state updates players move.